Makefile

```diff
@@ -67,6 +67,19 @@ test-gaia-e2e-parallel-tagged:
 test-no-cache:
 	go test ./... -count=1 && go run ./tests/e2e/...
 
+###############################################################################
+###                                Linting                                  ###
+###############################################################################
+
+lint:
+	@echo "--> Running linter"
+	golangci-lint run  ./... --config .golangci.yml
```

Comment:
Should make install a particular version of golangci-lint first? e.g.,  https://github.com/cosmos/cosmos-sdk/blob/58f3a4a2375f0b617ee0ac3399085c1f996195ba/Makefile#L387

Reply:
Yea I've added that here https://github.com/cosmos/interchain-security/pull/860/commits/032ce34f5096974c402b49e2b046e24b5c08ba3d, the version in your link didn't build for me, but I think it's that I still have go 1.18 installed. Might be worth waiting on #840